outputs.tf

```diff
@@ -38,3 +38,13 @@ output "cbr_rule_ids" {
   description = "CBR rule ids created to restrict Postgresql"
   value       = module.cbr_rule[*].rule_id
 }
+
+output "hostname" {
+  description = "Database hostname"
+  value       = nonsensitive(local.hostname)
+}
+
+output "port" {
+  description = "Database port"
+  value       = nonsensitive(local.port)
+}
```

Comment:
perhaps we should say these outputs will only have values if either service credentials or database users are created ?